helm/templates/pod_disruption_budget.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.podDisruptionBudget.enabled -}}
```

Comment:
let's add this to the values file

```
podDisruptionBudget:
  enabled: false
```

Reply:
Was added in #13, so I just updated this PR to have those new changes as well